string robot_id
int16 x_coord
int16 y_coord